CodeMaid/Logic/Cleaning/UsingStatementCleanupLogic.cs

```diff
@@ -73,16 +74,9 @@ public void RemoveAndSortUsingStatements(TextDocument textDocument)
             // Capture all existing using statements that should be re-inserted if removed.
             const string patternFormat = @"^[ \t]*{0}[ \t]*\r?\n";
 
-            var points = (from usingStatement in _usingStatementsToReinsertWhenRemoved.Value
-                          from editPoint in TextDocumentHelper.FindMatches(textDocument, string.Format(patternFormat, usingStatement))
-                          select new { editPoint, text = editPoint.GetLine() }).Reverse().ToList();
-
-            // Shift every captured point one character to the right so they will auto-advance
-            // during new insertions at the start of the line.
-            foreach (var point in points)
-            {
-                point.editPoint.CharRight();
-            }
+            var usingStatementsToReinsert = _usingStatementsToReinsertWhenRemoved.Value
```

Comment:
Previously I was trying to capture and re-insert text back where it was originally located.  This helped with some custom sort rules that users had.  I agree it may not be necessary anymore since newer versions of VS have merged remove and sort back together vs. being separate options.

Reply:
That makes sense, no problem.